cockatrice/src/qt-json/json.cpp

```diff
@@ -269,8 +269,7 @@ QVariant Json::parseObject(const QString &json, int &index, bool &success)
         Json::nextToken(json, index);
 
         //Loop through all of the key/value pairs of the object
-        bool done = false;
-        while(!done)
+        while(true)
```

Comment:
I changed it intentionally. The review site said that done was never changed to true anywhere in the code and I looked so I just changed it to true to shut them up